src/main/resources/application.properties

```diff
@@ -1,7 +1,10 @@
 spring.jpa.hibernate.ddl-auto=update
-spring.datasource.url=jdbc:mysql://localhost:3306/db?createDatabaseIfNotExist=true&useEncoding=true&characterEncoding=UTF-8&autoReconnect=true&useLegacyDatetimeCode=false&serverTimezone=UTC
+spring.datasource.url=jdbc:mysql://localhost:3306/actent?createDatabaseIfNotExist=true&useEncoding=true&characterEncoding=UTF-8&autoReconnect=true&useLegacyDatetimeCode=false&serverTimezone=UTC
 spring.datasource.username=root
-spring.datasource.password=root
+spring.datasource.password=java
 spring.jpa.show-sql=true
 server.port=8080
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
\ No newline at end of file
+
+
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
+
```

Comment:
Why are you pushing your properties file? Remove this from merge request.

Reply:
done

Reply:
You still trying to merge new properties file into develop. Unresolving.

Reply:
done